components/gallery/GalleryItemToolBar.vue

```diff
@@ -63,6 +63,13 @@ const mediaAndImageType = computed(() => {
   return { mediaType, imageType }
 })
 
+const determineElementType = (mediaType, imageType) => {
+  if ([MediaType.IFRAME, MediaType.VIDEO].includes(mediaType)) {
+    return mediaType
+  }
+  return imageType
+}
+
```

Comment:
please move logic to utils

Reply:
Sorry, I have been away. I will attend to this right away.